Added help section about nightly (#70)
Chatterino/wiki: docs/Help.md

2. Back up your settings by copying them from your [Chatterino folder](#where-is-chatterino-folder-located) to a safe location on your disk and delete them. Restart your computer and try launching Chatterino again.
3. Back up your settings, uninstall Chatterino and install it again, but make sure `Fresh Install` option is checked.
4. Try running Chatterino in portable mode.

### What is nightly and how to use / install it

##### Overview
Nightly is a "developer release" of Chatterino. It is released every time there's a new change added to source code. It includes some features that may not be available in the latest stable release.

##### Installation
1. Go to [nightly release page](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build) on GitHub.
1. Download nightly:
    - For **Windows** download `test-cmake-chatterino-windows-x86-64.zip`
    - For **Linux** download `test-cmake-Chatterino-x86_64.AppImage`
    - For **Mac** download `test-cmake-chatterino-osx.dmg`
1. Install nightly:

>     - On **Windows**, right click the archive > `Extract All` > `Extract`. Open the newly extracted folder and move the `Chatterino2` folder to `C:\Program Files\Chatterino` (or wherever your stable Chatterino is installed).

Reviewer: ```suggestion
    - On **Windows**, right click the archive > `Extract All` > `Extract` (Override files if prompted). Open the newly extracted folder and create a shortcut for the `chatterino.exe` file to the Desktop for easy access.
```

    - On **Linux**, open up the download directory in your terminal and execute the following command `chmod +x test-cmake-Chatterino-x86_64.AppImage && sudo mv test-cmake-Chatterino-x86_64.AppImage /usr/local/bin`
    - On **Mac**, open the dmg file and drag Chatterino into the Applications folder.

##### Updating

> Nightly ignores version checking and it will not attempt to update on its own. Because of that it will not notify you when a new stable update is available. To update, simply repeat steps above. If you want to switch back to stable release, simply download and install it from [Chatterino's website](https://chatterino.com/#downloads).

Reviewer: ```suggestion
Nightly ignores version checking and it will not attempt to update on its own. Because of that it will not notify you when a new stable update is available. To update, repeat steps above. If you want to switch back to stable release, download and install it from [Chatterino's website](https://chatterino.com/#downloads).
```

2. Back up your settings by copying them from your [Chatterino folder](#where-is-chatterino-folder-located) to a safe location on your disk and delete them. Restart your computer and try launching Chatterino again.
3. Back up your settings, uninstall Chatterino and install it again, but make sure `Fresh Install` option is checked.
4. Try running Chatterino in portable mode.

### What is nightly and how to use / install it

##### Overview
Nightly is a "developer release" of Chatterino. It is released every time there's a new change added to source code. It includes some features that may not be available in the latest stable release.

> 

Reviewer 2: ```suggestion
```

##### Installation
1. Go to [nightly release page](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build) on GitHub.
1. Download nightly:

>     - For **Windows** download `test-cmake-chatterino-windows-x86-64.zip`

Reviewer 2: ```suggestion
    - For **Windows**, download `test-cmake-chatterino-windows-x86-64.zip`
```

Author: Since those are very simple sentences I don't feel like comma is necessary in here 😕

Reviewer 3: Doubting the god native speaker?
![forsenSmug](https://user-images.githubusercontent.com/35232120/116544654-b444f400-a8ef-11eb-857c-841b32e1ebce.png)

Reviewer 2: Where a pause is taken in a sentence, you should add a comma.

>     - For **Linux** download `test-cmake-Chatterino-x86_64.AppImage`

Reviewer 2: ```suggestion
    - For **Linux**, download `test-cmake-Chatterino-x86_64.AppImage`
```

Author: Since those are very simple sentences I don't feel like comma is necessary in here 😕

>     - For **Mac** download `test-cmake-chatterino-osx.dmg`

Reviewer 2: ```suggestion
    - For **Mac**, download `test-cmake-chatterino-osx.dmg`
```

Author: Since those are very simple sentences I don't feel like comma is necessary in here 😕

1. Install nightly:

>     - On **Windows** extract it with 7zip / winrar and import files from extracted `Chatterino2` folder to `C:\Program Files\Chatterino` (or whereever is your application installed).

Reviewer 2: ```suggestion
    - On **Windows**, right click the archive > `Extract All` > `Extract`. Open the newly extracted folder and move the `Chatterino2` folder to `C:\Program Files\Chatterino` (or wherever is your application installed).
```

>     - On **Linux** open up download directory in your terminal and execute the following command `chmod +x test-cmake-Chatterino-x86_64.AppImage && sudo mv test-cmake-Chatterino-x86_64.AppImage /usr/local/bin`

Reviewer 2: ```suggestion
    - On **Linux**, open up the download directory in your terminal and execute the following command: `chmod +x test-cmake-Chatterino-x86_64.AppImage && sudo mv test-cmake-Chatterino-x86_64.AppImage /usr/local/bin`
```

Author: Why add a semicolon here and not everywhere else? Makes no sense to me.

Reviewer 2: It was added in the windows section also above, only mac section is left...

>     - On **Mac** (mac nerds please help me out)

Reviewer 4: Open the dmg and drag Chatterino into the Applications Folder

https://www.howtogeek.com/362166/what-is-a-dmg-file-and-how-do-i-use-one/

Author: Thank you very much, sir! ♥

>     - On **Mac** open the dmg file and drag Chatterino into the Applications folder.

Reviewer 2: ```suggestion
    - On **Mac**, open the dmg file and drag Chatterino into the Applications folder.
```
